src/storage/src/upsert_continual_feedback.rs

```diff
@@ -342,19 +341,31 @@ where
 
                                 stage_input(
                                     &mut stash,
-                                    cap,
                                     &mut data,
                                     &input_upper,
                                     &resume_upper,
                                 );
+                                if !stash.is_empty() {
+                                    // Update the stashed capability to the minimum
```

Comment:
If I understand correctly, this comment seems a little misleading because it's the "minimum" time which kind of means the "maximum" capability?

caveat: I definitely have a more limited understanding of timely, so feel free to ignore me here!

Reply:
It's true that the minimum capability is maximally flexible. I've always seen these adjectives refer to the contained time and not to the capability's flexibility though so I think "minimum" here matches the intended meaning of retaining the capability with the minimum time

Reply:
One option is to keep a `CapabilitySet` here instead... IIUC will do the equivalent minimum-finding, but maybe more explicit about intent? Would be fine as a followup in any case!